cli.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+var updater = require('update-notifier'),
+	pkg     = require('./package.json');
+
+updater({pkg: pkg}).notify();
+
+var yargs = require('yargs')
+	.usage('dynector: conveniences for interacting with the dynect API')
+	.option('zone',
+	{
+		alias: 'z',
+		description: 'specify a zone',
+	})
+	.option('replace',
+	{
+		alias: 'r',
+		describe: 'remove the node before acting',
+		type: 'boolean'
+	})
+	.option('silent',
+	{
+		describe: 'do not log informationally',
+		type: 'boolean'
+	})
+	.example('dynector arecord example.com 10.0.0.1')
```

Comment:
Examples would look slightly better in the terminal if each one had a brief description, as second argument to each `.example()` call.